tools/internal_ci/windows/grpc_basictests.cfg

```diff
@@ -0,0 +1,30 @@
+# Copyright 2017 gRPC authors.
```

Comment:
Just saw you deleted `grpc_master.cfg` for linux in favor for `grpc_basictests.cfg`. `grpc_master.cfg` can be deleted in a future PR, after updated in Kokoro.